java/tsfile/src/main/java/org/apache/tsfile/encrypt/EncryptUtils.java

```diff
@@ -106,78 +122,86 @@ public static String byteArrayToHexString(byte[] bytes) {
     return sb.toString();
   }
 
-  public static String getNormalKeyStr() {
+  public static String computeNormalKeyStr() {
+    final MessageDigest md;
     try {
-      MessageDigest md = MessageDigest.getInstance("SHA-256");
-      md.update("IoTDB is the best".getBytes());
-      md.update(TSFileDescriptor.getInstance().getConfig().getEncryptKey().getBytes());
-      byte[] data_key = Arrays.copyOfRange(md.digest(), 0, 16);
-      data_key =
-          IEncryptor.getEncryptor(
-                  TSFileDescriptor.getInstance().getConfig().getEncryptType(),
-                  TSFileDescriptor.getInstance().getConfig().getEncryptKey().getBytes())
-              .encrypt(data_key);
-
-      StringBuilder valueStr = new StringBuilder();
-
-      for (byte b : data_key) {
-        valueStr.append(b).append(",");
-      }
-
-      valueStr.deleteCharAt(valueStr.length() - 1);
-      String str = valueStr.toString();
-
-      return str;
-    } catch (Exception e) {
+      md = MessageDigest.getInstance("SHA-256");
+    } catch (NoSuchAlgorithmException e) {
       throw new EncryptException(
-          "SHA-256 function not found while using SHA-256 to generate data key", e);
+          "SHA-256 algorithm not found while using SHA-256 to generate data key", e);
     }
+    md.update("IoTDB is the best".getBytes());
+    md.update(TSFileDescriptor.getInstance().getConfig().getEncryptKey().getBytes());
+    byte[] data_key = Arrays.copyOfRange(md.digest(), 0, 16);
+    data_key =
+        IEncryptor.getEncryptor(
+                TSFileDescriptor.getInstance().getConfig().getEncryptType(),
+                TSFileDescriptor.getInstance().getConfig().getEncryptKey().getBytes())
+            .encrypt(data_key);
+
+    StringBuilder valueStr = new StringBuilder();
+
+    for (byte b : data_key) {
+      valueStr.append(b).append(",");
+    }
+
+    valueStr.deleteCharAt(valueStr.length() - 1);
+    return valueStr.toString();
   }
```

Comment:
What is the difference between this method and `getNormalKeyStr(TSFileDescriptor.getInstance().getConfig())`?